### PR TITLE
Added Yams Console/Windows service host to be used on ARM templates.

### PR DESCRIPTION
**Pull request by galvesribeiro**
This is the first step to a one-click deployment of Yams silo on VMs.

### CODE REVIEW

File: Samples/Etg.Yams.ARM/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## What is next? ##
+
+1. Create a Chocolatey package and publish it to chocolatey.org
+2. Create an Azure Resource Manager (ARM) Template
+3. Put a on-click-deply link on Yams repo so people will be able to deploy a Yams cluster by just click on this link and fill the wizard parameters.
+
+
+More news soon!
+
+[]s
```
Comment (ReubenBond) [inline Samples/Etg.Yams.ARM/README.md]: What's this last line?

Comment (galvesribeiro) [inline Samples/Etg.Yams.ARM/README.md]: Sorry... Old habit... It means `hugs` :)

Will remove on next PR which will update this file no worries

Comment (ReubenBond) [inline Samples/Etg.Yams.ARM/README.md]: haha, cool - I just thought they were some stray chars :P keep it

**Comment by nehmebilal** [comment]
Looks good, thanks @galvesribeiro!

**Comment by galvesribeiro** [comment]
Thanks!

**Comment by sergeybykov** [comment]
First 'external' PR merged to YAMS!

**Comment by galvesribeiro** [comment]
lol

**Pull request merged**